docs/operations/metrics.md

```diff
@@ -326,6 +326,12 @@ If `emitBalancingStats` is set to `true` in the Coordinator [dynamic configurati
 
 ## General Health
 
+### Service Health
+
+|Metric|Description|Dimensions|Normal Value|
+|------|-----------|----------|------------|
+|`druid/heartbeat`| Report service health. For Overlord/Coordinator, the dimension is leader count. `ServiceStatusMonitor` must be enabled. |`heartbeatType`|1|
```

Comment:
Nit: I think we should prefix this metric with either `cluster/` or `server/` so that this metric becomes `cluster/heartbeat` or `server/heartbeat`. We can add other relevant metrics later which have the same prefix. 

The prefix `druid/` doesn't seem to give any info about the metric.
@suneet-s , @YongGang , what do you think?

Reply:
I don't have a strong preference. We could also just call the metric `heartbeat` instead of `druid/hearbeat`

The Druid process might be running in a container, in which case `server` can be misleading. Do you have some examples in mind of other metrics that would live under `cluster/` that wouldn't make sense under `druid/`?

Reply:
No, I don't have a concrete example in mind either. I was thinking mostly of any cluster-level information, inter-service communication, etc.

`druid/` is pretty much a catch-all and any metric that goes under `cluster/` could potentially go under `druid/` as well. But I generally try to use prefixes that make the metrics a little more user-friendly and somewhat self-explanatory.

I agree with you that `server/` can be misleading when running on containers. I avoided using it in a PR due to similar reasons.

I am working on a PR where I think I am going to use `cluster/` for server view syncs. e.g. `cluster/serverview/synced` which denotes the sync status between coordinator/broker inventory and different historical/peon processes.

Reply:
`service/` prefix seems fine to me as the heartbeat doesn't have to be on `cluster` level.